redis/1.11/README.md

```diff
@@ -0,0 +1,179 @@
+# How to use Redis on DC/OS
+
+[Redis](http://redis.io/) is a popular in-memory data structure store, used as database, cache and message broker.
+
+- Estimated time for completion: 5 minutes
+- Target audience: Anyone interested using an in-memory data store.
+- Scope: Learn how to use Redis on DC/OS
+
+Note, that there are two different DC/OS services for Redis:
+- Redis: A single Redis container, easy to install and test but not suited for production workloads.
+- mr-redis: A framework allowing you to mange multiple Redis datastores standalone or in a clustered setup.
+
+**Table of Contents**:
+
+
+- [Prerequisites](#prerequisites)
+- [Use single container Redis service](#single-container-redis)
+- [mr-redis framework](#mr-redis)
+- [Install mr-redis](#install-mr-redis)
+- [Create a Redis instance](#create-a-redis-instance)
+- [Access the Redis instance](##access-the-redis-instance)
+- [Uninstall mr-redis](#uninstall-mr-redis)
+
+## Prerequisites
+
+- A running DC/OS 1.11 cluster with at least 2 nodes with 1 CPU and 300MB of RAM available.
+- [DC/OS CLI](https://docs.mesosphere.com/1.11/cli/install/) installed.
+
+## Single container Redis
+Note, that this service is targeted at testing Redis and not for production workloads!
+You can install the single Redis container with a simple `dcos package install redis`.
+Then the easiest option to test Redis is by storing a key manually via the redis-cli
+- SSH into node where Redis is running: 
+``` bash 
+dcos node ssh --master-proxy --mesos-id=$(dcos task redis --json | jq -r '.[] | .slave_id')
+```
+NOTE: This requires you to have the ssh-key required to conenct to the machines added to your local ssh agent (e.g., via ssh-add my_public_key).
+- Because Redis is running in docker container, we can list all docker containers using docker ps and get the ContainerID.
+- Connect to a bash session to the running container: `sudo docker exec -it CONTAINER_ID /bin/bash`
+- Start the Redis CLI: redis-cli
+  - Set key i`set mykey key1`
```

Comment:
I'd like to see the formatting improved here, the lack of space between "i" and "set" here makes it a bit difficult to read. Maybe try some nested bullets, something like this for this line and the next?
 - Set key i:
   - `set mykey key1`